release-notes/3.0/preview/api-diff/preview6/standalone-packages/3.0-preview6-standalone-packages.md

```diff
@@ -0,0 +1,13 @@
+# API Difference E:\code\Validation\p5OOBs vs E:\code\Validation\p6OOBs
```

Comment:
The title needs to be updated to match previous diffs:
https://github.com/dotnet/core/blob/master/release-notes/3.0/preview/api-diff/preview4/standalone-packages/3.0-preview4-standalone-packages.md#api-difference-standalone-preview3-vs-standalone-preview4

Reply:
Whoops, thanks for noticing - fixed